src/main/java/com/hrznstudio/titanium/util/TitaniumRegistries.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * This file is part of Titanium
+ * Copyright (C) 2020, Horizon Studio <contact@hrznstudio.com>.
+ *
+ * This code is licensed under GNU Lesser General Public License v3.0, the full license text can be found in LICENSE.txt
+ */
+
+package com.hrznstudio.titanium.util;
+
+public class TitaniumRegistries {
```

Comment:
It's empyt, but still here?

Reply:
Not sure if this will be used in the end to store registry references.
But that was the plan before.